packages/core/src/libraries/quota.ts

```diff
@@ -55,6 +92,10 @@ export const createQuotaLibrary = (queries: Queries, cloudConnection: CloudConne
     const plan = await getTenantSubscriptionPlan(cloudConnection);
```

Comment:
nit: feels like setTenantSubscriptionPlan can extract as a middleware, and provides tenant subscription status in the context.  Could do later.